__test__/api/applications/createSingleApplication.test.js

```diff
@@ -1,35 +1,34 @@
 const app = require("../../../server");
 const supertest = require("supertest");
-const mongoose = require("mongoose");
 //const applicationHandler = require("../../../utils/responseHandler");
 const { getOrgToken } = require("../../../utils/auth/tokenGenerator");
 const request = supertest(app);
 
 describe("POST /applications", () => {
-    let orgToken, url;
-    let invalidToken = "5f08err3e9dced1ad07err48b9b57a";
+  let orgToken, url;
+  let invalidToken = "5f08err3e9dced1ad07err48b9b57a";
 
-    beforeAll(async () => {
-        orgToken = await getOrgToken(global.organization._id, global.admin._id);
-        url = "/v1/applications";
-    });
-  
-    it("Should successfully create an application", async () => {
-      let res = await request
-        .post(url)
-        .set("Authorization", `bearer ${orgToken}`)
-        .send({ name: "test_app" });
-      expect(res.status).toEqual(201);
-      expect(res.body.status).toEqual("success");
-    });
+  beforeAll(async () => {
+    orgToken = await getOrgToken(global.organization._id, global.admin._id);
+    url = "/v1/applications";
+  });
 
-    it("Should return a 401 error for invalid Token", async () => {
-        let res = await request
-          .post(url)
-          .set("Authorization", `bearer ${invalidToken}`)
-          .send({ name: "test_app" });
-  
-        expect(res.status).toEqual(401);
-        expect(res.body.status).toEqual("error");
-      });
-});
\ No newline at end of file
+  it("Should successfully create an application", async () => {
+    let res = await request
+      .post(url)
+      .set("Authorization", `bearer ${orgToken}`)
+      .send({ name: "test_app" });
+    expect(res.status).toEqual(201);
+    expect(res.body.status).toEqual("success");
+  });

```

Comment:
You're not testing for the values returned .. please expect them, refer to the controller and see the data returned

Reply:
Thanks. Changes made

Reply:
You need to also test for validationErrors (error 4220

Reply:
You need to also test for input validationErrors (error 422)